src/main/ngapp/accio/src/app/home/home.component.ts

```diff
@@ -1,15 +1,67 @@
 import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import * as $ from 'jquery';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css']
 })
 export class HomeComponent implements OnInit {
+  // TODO: Add filter global once talk w/ DB team to understand it.
+  //       Also add it as attribute in getProject's url var.
+  filterVisibility: string;
+  filterRole: string;
+  filterTag: string;
+  filterId: string;
+  filterSort: string;
 
-  constructor() { }
+  projectPath: string;
+  projects: Array<any>;
+
+  constructor(
+      private router: Router) { }
 
   ngOnInit(): void {
+    this.handleLoggedOut();
+  }
+  
+  // Redirects user to intro page if not logged in
+  async handleLoggedOut(): Promise<void> {
+    console.log('handling login...');
+    const response = await fetch('/login-status?page=');
+    /**Content received contains 
+     * {loggedIn: boolean,
+     * url: string} 
+     */
```

Comment:
I'd suggest not using the `/** ... */` style of comment unless you are making JSDoc comments. For general comments, use `//` or `/* ... */`.